GooglePlayInstant/Samples/SphereBlast/scripts/AnimateLoadingText.cs

```diff
@@ -0,0 +1,65 @@
+﻿// Copyright 2018 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AnimateLoadingText : MonoBehaviour
```

Comment:
A similar capability will be provided by QuickDeploy (a loading bar). Not sure we want to duplicate it here.

Reply:
I'll keep as is for now, but will use your code when it's ready